### ISSUE TITLE
Inconsistent Community / Satellite event labeling

### ISSUE THREAD

**Issue by shua123** [chihacknight/chihacknight.org#148]
On the home page we are calling community/satellite events a "Community Event."  The template, tags, and url call them "satellite" events. Should we decide on one of these names and update the code for consistency?

**Comment by derekeder** [comment]
I think these started out being labeled as 'satellite events' but i found that it implies a hierarchy that i'd rather not encourage. I would like to call them 'community events' going forward.

**Comment by shua123** [comment]
Yeah, I like community events. The one potential conflict there is that we have tagged some regular hack nights with community (community feedback sessions, the Hackies, lightning talks). I don't know if we need a better way to distinguish between internal/chihacknight community and the larger Chicago community.
